Question about survey demographics (#731)
oddbird/oddleventy: content/blog/2024/css-demographics.md

---
title: What do survey demographics tell us?
sub: Are we measuring what we meant to measure?
date: 2024-10-31
author: miriam
tags:
  - CSS
summary: |
  There's been a lot of interest
  in the results of the annual
  State of CSS survey,
  but are we asking all the right questions?
---

Sacha Greif's annual
[State of CSS survey](https://2024.stateofcss.com/en-US/)
wrapped up recently,

> and the results are online.

Reviewer: just opinion, but I like `available` here :) optional suggestion:

```suggestion
and the results are available.
```

I've seen a number of people

> commenting on the

Reviewer: Do we want to link to any of these existing discussions?

- Is CSS mostly used for web apps?
- Does the industry skew young to the same degree as the survey?

If you don't know [Kevin Powell's work teaching CSS](https://www.kevinpowell.co/),
I highly recommend it.
But as you might expect,
his audience is weighted towards students
and people starting out in the field.
If he's a [primary source for respondents](https://2024.stateofcss.com/en-US/demographics/#source),
that's likely to impact the demographics.

And it's great to get that perspective!
But it still requires us to ask:
are we looking at data about CSS authors generally,
or are we looking at a small slice of the industry?
And is that slice more broadly representative?

In many cases,
polls use demographics data to
_adjust for response bias_ --
but that requires an understanding of how the demographics
interact with other data points.
How would it change our interpretation of the survey more generally
if we knew the answers to those questions?

- Do most CSS authors test their designs only on desktop,
  or is that specific to people who write CSS on the side?
- Is Tailwind broadly popular,
  or within certain demographics?

> - What other questions might rise up from this analysis?

Reviewer: ```suggestion
- What other questions might rise up from this analysis?
- Why did Winging It not make it to the top of the Video Creators list?
```

https://2024.stateofcss.com/en-US/resources/#video_creators

> - What other questions might rise up from this analysis?

Reviewer 2: ```suggestion
- What other questions might arise from this analysis?
```

I don't have answers here.
I'm just not sure what I'm looking at,

> or how it should be read.

Reviewer: Can we end with a CTA? Maybe asking what we missed or a poll or something to see if our readers filled out that survey (I didn't). 

I am ALWAYS critiquing surveys for the questions they didn't ask, or for the ways in which I would have worded something differently. I'm not saying anything about this survey especially since I didn't take it, but I tend to have an issue with many surveys I see.

If you don't know [Kevin Powell's work teaching CSS](https://www.kevinpowell.co/),
I highly recommend it.

> But we you might expect

Reviewer 2: ```suggestion
But as you might expect
```

his audience is weighted towards students.
If he's a [primary source for respondents](https://2024.stateofcss.com/en-US/demographics/#source),
that's likely to impact the demographics, right?

There's no problem with that --
student perspectives are extremely useful! --
but it might change how we read the data.
With under 10,000 responses,
we know we're looking at a small
and self-selected sample of the field.
I know several excellent CSS authors
who didn't take the survey.
Are some more likely to respond than others?
For what reasons?
Are these results representative of CSS authors generally?
How would we know?

The largest 'source' is the survey itself,
which likely represents repeat participants.
So where did they come from initially?
The earliest data I can find is
[from the 2019 survey](https://2019.stateofcss.com/demographics/#source),

> with the **State of JS**

Reviewer 2: ```suggestion
with the **State of JS** survey
```

as the most popular source.

> maybe it's not surprising if a majority of those respondents

Reviewer 2: ```suggestion
Maybe it's not surprising if a majority of those respondents
```

write JS as their primary job?

In many cases,
polls use demographic data
_to adjust for response bias_ --
but that requires an understanding of how well
the demographics match the larger reality,
and also how that is likely to impact other data points.
How would it change our interpretation of the survey more generally
if we knew the answers to those questions?

- Do most CSS authors test their designs only on desktop,
  or is that specific to people who write CSS on the side?
- Is Tailwind broadly popular,
  or specifically within certain demographics?
- What other questions might arise from this analysis?
- Why aren't the OddBird [Winging It](https://www.oddbird.net/wingingit/) videos your top resource? 😅

The survey does allow us
to break down the data in various ways --
and run comparisons.
I don't feel qualified to parse out
what correlations are significant or relevant.
But poking around a bit,
a few things jumped out out:

> - Men seem more likely to write JS primarily,
>   while women are more likely to write CSS primarily
> - Respondents who write more JS are
>   less likely to test across environments
>   (and also less likely to answer that question)

Reviewer 2: ```suggestion
- Men seem more likely to write JS primarily,
  while women are more likely to write CSS primarily.
- Respondents who write more JS are
  less likely to test across environments
  (and also less likely to answer that question).
```

- Kevin Powell sent people in all age brackets,
  but his largest bracket (22%) was people over 60!
  We love to see it.
  His next bracket (18%) was under 20.

>   I don't know why I'm doing Kevin Analytics,

Reviewer 2: I'm not sure if this is missing some words, or a typo?

```suggestion
```
